pegasus/sites.v3/code.org/public/educate/resources/recruit.md

```diff
@@ -105,7 +105,7 @@ Hang these posters in the halls along with the course number for your computer s
 *Learn more about these young tech professionals at [code.org/careers](/careers-in-tech)*
 
 [![image](/images/fit-150/malala.png)](/files/malala-poster.pdf)
-[![image](/images/fit-150/obama-new.png)](/files/obama-poster-new.pdf)
+[![image](/images/fit-150/obama-new-old.png)](/files/obama-poster-new.pdf)
```

Comment:
!

Reply:
I know. I didn't rename it! haha

![giphy](https://user-images.githubusercontent.com/1920530/47472003-c810bc00-d7c1-11e8-906c-616bd97e7004.gif)